README.md

```diff
@@ -37,6 +32,7 @@ In the framework of this project where trained two famous NN's - Inceptionv3 and
 During project where used following libraries:
 
 - [PyTorch] - Deep Learning Framework
+- [PyTorch Lightning](https://github.com/PyTorchLightning/pytorch-lightning) - The lightweight PyTorch wrapper for high-performance AI research. Scale your models, not the boilerplate.
```

Comment:
Please try to use Pytorch Lightning or Catalyst for regular training. The code would be much cleaner and alike and more engineer could support it

Reply:
Thank you for your suggestions. I pushed updated files in the repo. Please check